tutorials/10_async.md

```diff
@@ -1,13 +1,43 @@
-Generátory a AsyncIO
-====================
+# Generátory a AsyncIO
 
 Výukové materiály:
 [naucse.python.cz](http://naucse.python.cz/2017/mipyt-zima/intro/async/),
 [GitHub](https://github.com/pyvec/naucse.python.cz/tree/master/lessons/intro/async).
 
-Úkol
-====
+## Úkol
 
-Vaším úkolem za 5 bodů bude předělat první úlohu pomocí asyncio.
+Vaším úkolem za 5 bodů je implementovat funkci `gather_acquaintances`, která pomocí asynchronní komunikace s [GitHub API] (včetně [GitHub Search API]) ze všech* issues podle zadané specifikace zjistí všechny uživatele, kteří tyto issue komentovali, a to včetně počtu komentářů. Tato funkce nebude mít asynchronní API (tedy nebude `async def`, ale jen `def`).
 
-Zadání ještě není hotovo.
+*) [GitHub Search API] má limit na 1000 výsledků
+
+Funkce `asya.logic.gather_acquaintances` (v projektu [MarekSuchanek/asya]):
+
+* parametr `search_specs` = slovník obsahující specifikaci vyhledávání issues pro [GitHub Search API] (možné rovnou používat pro požadavky)
+* parametr `supervisor` = objekt, na kterém vaše implementace volá příslušné metody v daných částech kódu (třídu `AsyaSupervisor` není možné měnit, ale lze "registrovat" vlastní funkce, které se budou volat, není to však vyžadováno - viz [dokumentace třídy](http://asya.readthedocs.io/en/latest/api.html#module-asya.supervisor))
```

Comment:
Chápu dobře, že kód studenta *musí* volat metody `register_*` podle toho, co je u nich napsáno v dokumentaci? To by asi mělo být řečeno explicitněji.

Reply:
https://github.com/cvut/MI-PYT/pull/123/commits/2f73fe4ff60cee49f0c40fba0d1c672a7a524264